### ISSUE TITLE
Parameters for more subtle output effect

### ISSUE THREAD

**Issue by jakeelwes** [idealo/image-super-resolution#53]
Just wondering if there are any parameters I can play with to get a slightly less intense brushy effect? The examples look a bit more realistic/less like a filter effect than mine, if that makes any sense?
<img width="1680" alt="Screen Shot 2019-08-05 at 23 06 54" src="https://user-images.githubusercontent.com/10772693/62498145-1e416f80-b7d6-11e9-9306-f499c060c267.png">

**Comment by cfrancesco** [comment]
This is an effect of the noise cancelling model we provide, there is a few things you could do:
- re-train the current model, with some data that is similar to the images you want to upscale (maybe the images themselves?)
- take two or more model that do different things and merge the weights (seems like a feature to add asap)
- or just use the gans-RRDN model if you don't need to remove jpeg artifacts

**Comment by jakeelwes** [comment]
Ahh cool, thanks @cfrancesco I'll let you know how I get on :)

**Comment by jakeelwes** [comment]
Yeah I ran it through a combination of the noise cancelling and non noise cancelling models which gave a better result, would be cool to merge the weights!
